src/lib/sidebar/About.svelte

```diff
@@ -6,67 +6,71 @@
 </script>
 
 <Modal title="About the Active Travel Infrastructure Platform" bind:open>
-  <p>
-    ATIP v2 is an
-    <ExternalLink href="https://github.com/acteng/atip"
-      >open source project</ExternalLink
-    > supported by Active Travel England and developed by:
-  </p>
+  <div class="govuk-prose">
```

Comment:
Here's an example of auto-applying CSS classes to everything below. All the diff below is just introducing one level of nesting in the DOM